index.html

```diff
@@ -88,49 +94,241 @@
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
-  var options = {
+    var options = {
     yearRange: 'all'
   };
+  var svg, x, y, xAxis, yAxis, xh, yh; 
+  var margin = { top: 15, right: 15, bottom: 30, left: 45 };
```

Comment:
So up here you are declaring 'yh' as a variable, but later on in the code you refer to 'hy' which is undeclared.